tests/runner/mock_host.py

```
@@ -22,7 +22,9 @@
 import pickle
 import io
 
-from base_host import *
+from origin.base_host import *
```

Ruff (0.13.3)

`from origin.base_host import *` used; unable to detect undefined names

(F403)